program/databases/db_tests

```diff
@@ -6986,3 +6985,13 @@
 "007325","","8","/radio.php","GET","Login Page","","password","","","PHP backdoor found.","",""
 "007326","","1","/app-ads.txt","GET","DIRECT","google.com,","","","","The ads.txt file can reveal information about Google Adwords or other advertising networks in use.","",""
 "007327","RFC-5785","3","/.well-known/jwks","GET","keys","sig","","","","JWKS file found","",""
+"007328","htaccess","3","/.htaccess","GET","200","","(Options|Require|AddDefaultCharset|AuthType|AuthName|AuthUserFile|Deny|Allow|Redirect|RewriteBase|RewriteCond|RewriteRule|DirectoryIndex)","","","","",""
+"007329","setup_php","c","/setup.php","GET","200","","<\?php.*\$db_host.*\$db_name.*\$db_user.*\$db_password.*new\s+PDO\(.*CREATE\s+TABLE\s+IF\s+NOT\s+EXISTS","","","","",""
+"007330","nginx","3","/nginx.conf","GET","200","","^\s*server\s*{(?:[^{}]+|(?R))*\s*}","","","","",""
+"007331","instructions_php","3","/instructions.php","GET","200","","\b(?:Instructions|Download|Installation|Install)\b","","","","",""
+"007332","phpinfo","3","/phpinfo.php","GET","200","","<?php\s*phpinfo\(\);\s*\?>","\b(PHP Version|System|Build Date|Build System)\b","","","","","",""
+"007333","NEW001","","/php.ini","GET","200","","= (Off|On|off|on)\s*","","","The php.ini may contain important php settings.","","","","",""
+"007334","NEW002","","/phpinfo.php","GET","200","",">PHP Version <\/td><td class="v">([0-9.]+)","","","phpinfo() exposes information about the configuration of the PHP environment and server.","","","","",""
+"007335","README","3","/README.*.md","GET","200","","^#\s.*\n\n## Description\n.*\n\n## Installation\n.*\n\n## Usage\n.*\n\n## Contributing\n.*\n\n## License\n.*","","","","",""
+"007336","config_json","3","/config.json","GET","200","","appSettings":\s*\{\s*"logLevel":\s*".*?",\s*"maxConnections":\s*\d+,\s*"timeout":\s*\d+\s*\}","","","","",""
```

Comment:
this one seems like it should be general as well, we don't detect it on brokencrystals. I think it's a config for react but i'm not sure.